src/config/src/config.erl

```diff
@@ -66,17 +71,17 @@ start_link(IniFilesDirs) ->
     gen_server:start_link({local, ?MODULE}, ?MODULE, IniFilesDirs, []).
 
 stop() ->
-    gen_server:cast(?MODULE, stop).
+    gen_server:stop(?MODULE).
```

Comment:
There is a proper `gen_sever:stop/1` now, so we can use that https://www.erlang.org/doc/apps/stdlib/gen_server.html#stop/1